<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>The Activated Sludge of Danish Wastewater Treatment Plants</title>
  <meta name="description" content="The Activated Sludge of Danish Wastewater Treatment Plants">
  <meta name="generator" content="bookdown 0.3.19 and GitBook 2.6.7">

  <meta property="og:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  
  
  

<meta name="author" content="Kasper Skytte Andersen">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  

<link rel="next" href="2-ordination.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-0.8/htmlwidgets.js"></script>
<link href="libs/plotlyjs-1.26.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.26.1/plotly-latest.min.js"></script>
<script src="libs/plotly-binding-4.6.0/plotly.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#the-history-of-wastewater-treatment-and-the-activated-sludge-process"><i class="fa fa-check"></i><b>1.1</b> The history of wastewater treatment and the activated sludge process</a></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#identification-of-microorganisms-in-activated-sludge"><i class="fa fa-check"></i><b>1.2</b> Identification of microorganisms in activated sludge</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#purpose"><i class="fa fa-check"></i><b>1.3</b> Purpose</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-ordination.html"><a href="2-ordination.html"><i class="fa fa-check"></i><b>2</b> Ordination in Microbial Ecology</a><ul>
<li class="chapter" data-level="2.1" data-path="2-ordination.html"><a href="2-ordination.html#exploratoryvsexplanatory"><i class="fa fa-check"></i><b>2.1</b> Exploratory vs.Â Explanatory</a></li>
<li class="chapter" data-level="2.2" data-path="2-ordination.html"><a href="2-ordination.html#nichedoublezero"><i class="fa fa-check"></i><b>2.2</b> Niche theory and the double-zero problem</a></li>
<li class="chapter" data-level="2.3" data-path="2-ordination.html"><a href="2-ordination.html#distance-based-ordination"><i class="fa fa-check"></i><b>2.3</b> Distance-based ordination</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-ordination.html"><a href="2-ordination.html#principal-coordinates-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Principal Coordinates Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-ordination.html"><a href="2-ordination.html#non-metric-multidimensional-scaling"><i class="fa fa-check"></i><b>2.3.2</b> non-Metric Multidimensional Scaling</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-ordination.html"><a href="2-ordination.html#metrics"><i class="fa fa-check"></i><b>2.3.3</b> Distance- and (dis)similarity measures</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-ordination.html"><a href="2-ordination.html#eigenanalysis-based-ordination"><i class="fa fa-check"></i><b>2.4</b> Eigenanalysis-based ordination</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-ordination.html"><a href="2-ordination.html#PCA"><i class="fa fa-check"></i><b>2.4.1</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-ordination.html"><a href="2-ordination.html#redundancy-analysis"><i class="fa fa-check"></i><b>2.4.2</b> Redundancy Analysis</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-ordination.html"><a href="2-ordination.html#CA"><i class="fa fa-check"></i><b>2.4.3</b> Correspondence Analysis and Canonical Correspondence Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims</a></li>
<li class="chapter" data-level="3" data-path="3-mm.html"><a href="3-mm.html"><i class="fa fa-check"></i><b>3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="3.1" data-path="3-mm.html"><a href="3-mm.html#sampling"><i class="fa fa-check"></i><b>3.1</b> Sampling</a></li>
<li class="chapter" data-level="3.2" data-path="3-mm.html"><a href="3-mm.html#dna-extraction"><i class="fa fa-check"></i><b>3.2</b> DNA extraction</a></li>
<li class="chapter" data-level="3.3" data-path="3-mm.html"><a href="3-mm.html#library-preparation-purification-and-pooling"><i class="fa fa-check"></i><b>3.3</b> Library preparation, purification and pooling</a></li>
<li class="chapter" data-level="3.4" data-path="3-mm.html"><a href="3-mm.html#dna-sequencing-and-bioinformatics"><i class="fa fa-check"></i><b>3.4</b> DNA sequencing and bioinformatics</a></li>
<li class="chapter" data-level="3.5" data-path="3-mm.html"><a href="3-mm.html#data-processing-and-analysis"><i class="fa fa-check"></i><b>3.5</b> Data processing and analysis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-results1.html"><a href="4-results1.html"><i class="fa fa-check"></i><b>4</b> Exploring the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="4.1" data-path="4-results1.html"><a href="4-results1.html#overview-of-the-differences"><i class="fa fa-check"></i><b>4.1</b> Overview of the differences</a></li>
<li class="chapter" data-level="4.2" data-path="4-results1.html"><a href="4-results1.html#wwtpcommunities"><i class="fa fa-check"></i><b>4.2</b> How does the microbial community composition describe the WWTPs?</a></li>
<li class="chapter" data-level="4.3" data-path="4-results1.html"><a href="4-results1.html#concluding-remarks"><i class="fa fa-check"></i><b>4.3</b> Concluding remarks</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-results2.html"><a href="5-results2.html"><i class="fa fa-check"></i><b>5</b> Explaining the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="5.1" data-path="5-results2.html"><a href="5-results2.html#the-influence-of-plant-design-on-the-microbial-communities"><i class="fa fa-check"></i><b>5.1</b> The influence of plant design on the microbial communities</a></li>
<li class="chapter" data-level="5.2" data-path="5-results2.html"><a href="5-results2.html#general-differences-related-to-sampling-time"><i class="fa fa-check"></i><b>5.2</b> General differences related to sampling time</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-discussion.html"><a href="6-discussion.html"><i class="fa fa-check"></i><b>6</b> General Discussion</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplementaryplots.html"><a href="A-supplementaryplots.html"><i class="fa fa-check"></i><b>A</b> Supplementary plots</a></li>
<li class="chapter" data-level="B" data-path="B-screeplots.html"><a href="B-screeplots.html"><i class="fa fa-check"></i><b>B</b> Scree plots</a><ul>
<li class="chapter" data-level="B.1" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpca"><i class="fa fa-check"></i><b>B.1</b> Scree plot of Figure @ref(fig:overviewPCA)</a></li>
<li class="chapter" data-level="B.2" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpcoabray"><i class="fa fa-check"></i><b>B.2</b> Scree plot of Figure @ref(fig:overviewPCoAbray)</a></li>
<li class="chapter" data-level="B.3" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewcca"><i class="fa fa-check"></i><b>B.3</b> Scree plot of Figure @ref(fig:overviewCCA)</a></li>
<li class="chapter" data-level="B.4" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigfactorccaa-d"><i class="fa fa-check"></i><b>B.4</b> Scree plot of Figure @ref(fig:factorCCA)(A-D)</a></li>
<li class="chapter" data-level="B.5" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigtimeccaab"><i class="fa fa-check"></i><b>B.5</b> Scree plot of Figure @ref(fig:timeCCA)(A+B)</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-WWTPcharacteristics.html"><a href="C-WWTPcharacteristics.html"><i class="fa fa-check"></i><b>C</b> Characteristics of the WWTPs</a></li>
<li class="chapter" data-level="D" data-path="D-references.html"><a href="D-references.html"><i class="fa fa-check"></i><b>D</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Activated Sludge of Danish Wastewater Treatment Plants</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">The Activated Sludge of Danish Wastewater Treatment Plants</h1>
<h4 class="author"><em>Kasper Skytte Andersen</em></h4>
<h4 class="date"><em><ol start="31" style="list-style-type: decimal">
<li>May 2017</li>
</ol></em></h4>
</div>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<div id="the-history-of-wastewater-treatment-and-the-activated-sludge-process" class="section level2">
<h2><span class="header-section-number">1.1</span> The history of wastewater treatment and the activated sludge process</h2>
<p>Cleaning wastewater is today an essential part of most modern civilised cities <span class="citation">(Orhon, 2015)</span>. However, the practice of cleaning wastewater has only been performed within the last century or so. In fact, until the turn of the 19th century, most cities did not even have a proper disposal system for sewage, it was simply poured onto the streets <span class="citation">(Orhon, 2015)</span>. It was first in the 1840s that the english lawyer Edwin Chadwick encouraged people to take âsanitary responsibilityâ and find a solution to the many problems associated with having sewage floating on the streets <span class="citation">(Hamlin, 1988)</span>. A few decades later, in the 1870s, the idea of leading the sewage through underground pipes called âsewersâ was widely adopted. The sewage was then primarily disposed of in one of two ways through irrigation; either it was led to the lands nearby, where farmers used it as a fertiliser, or it was distributed below the surface and penetrated into the soil in large sewage farms near the cities. Any remaining wastewater was simply led into nearby rivers, lakes or oceans, leading to eutrophication and pollution of the water bodies, which are major reasons why proper wastewater treatment is important <span class="citation">(Orhon, 2015)</span>.</p>
<p>In the early 1890s, sewage disposal was revolutionised by biological treatment of the sewage using contact filters invented by William Joseph Dibdin <span class="citation">(Hamlin, 1988)</span>. It was then possible to facilitate âpurificationâ of the sewage by immobilising naturally occuring bacteria in aeration tanks, which seemed to oxidise and destroy much of the sewage. During the next few decades, remarkable research in the fundamental understanding of wastewater treatment was achieved. The study of a âcompact brown growthâ suspended on filters or slates in aeration tanks was particularly investigated, as it was often associated with the removal of suspended solids and shorter aeration times <span class="citation">(Clark &amp; Adams, 1914)</span>. In 1914, Dr.Â Fowler wanted to test an idea by doing some âexperiments on the oxidation of sewage without the aid of filtersâ, which were carried out by his graduate engineers Edward Ardern and William Lockett <span class="citation">(Ardern &amp; Lockett, 1914)</span>. These pioneering experiments were to set a milestone in wastewater treatment with the discovery of what they called <em>Activated Sludge</em> (AS). By reusing the biologically active âcompact brown growthâ, which seemed to accumulate during sewage treatment, they were able to drastically improve the rate with which the sewage was treated. They simply performed lab-scale aerated batch experiments with sewage from the Manchester sewage treatment plant, where the accumulated AS was collected after âcomplete oxidation of the sewageâ (the term used at the time to describe the complete removal of organic matter from the water) and added to subsequent batches of sewage. They observed that the time needed for complete oxidation decreased with every batch, from weeks in the initial batches to less than a day in the final batches. These simple experiments founded the <em>Activated Sludge Process</em> (ASP), where the accumulated sludge is continously reused, which ever since has been the backbone of most wastewater treatment plants (WWTPs) in the world <span class="citation">(M. C. van Loosdrecht, Nielsen, Lopez-Vazquez, &amp; Brdjanovic, 2016)</span>.</p>
</div>
<div id="identification-of-microorganisms-in-activated-sludge" class="section level2">
<h2><span class="header-section-number">1.2</span> Identification of microorganisms in activated sludge</h2>
<p>Even a century ago, when Ardern and Lockett made their groundbreaking discovery, it was well known that microorganisms played a vital role in the processes involved in the purification of wastewater <span class="citation">(Fuller, 1915; P. H. Nielsen &amp; McMahon, 2014)</span>. At the time, however, little was known about which microorganisms were present in the AS or their particular role in wastewater treatment. The stability of operation and performance of early WWTPs established in the 1930-1940s were often unstable and many problems were encountered with e.g.Â foaming and bulking <span class="citation">(Orhon, 2015)</span>. Research into the microbiology of AS was therefore important, but the experimental methods available at the time to identify the microorganisms were limited, and research was primarily based on biochemical characterisation by isolation and cultivation, or morphological characterisation by light microscopy <span class="citation">(M. C. van Loosdrecht et al., 2016)</span>. Wastewater treatment was in the first half of the 20th century a highly empirical practice based on trial-and-error, and it was first in the early 1970s that significant research into the identity of problem-causing bacteria was done <span class="citation">(Eikelboom, 1975; M. C. van Loosdrecht et al., 2016)</span>.</p>
<p>Until now, much research has been done into linking the identity of microorganisms present in AS to their metabolic functions relevant to wastewater treatment. The combination of the development of molecular tools, e.g. <em>Polymerase Chain Reaction (PCR)</em> <span class="citation">(Mullis et al., 1986)</span>, various fluorescent oligonucleotide probes <span class="citation">(Amann et al., 1990; DeLong, Wickham, &amp; Pace, 1989)</span>, as well as DNA sequencing technologies have allowed for precise identification of the microorganisms present in the AS. This allows for a complete picture of the microbial community and has become invaluable information to the understanding of wastewater treatment. Today, the traditional <em>Sanger DNA sequencing</em> method developed in the late 1970s <span class="citation">(Sanger &amp; Coulson, 1975; Sanger, Nicklen, &amp; Coulson, 1977)</span> has been gradually replaced by high-throughput <em>next generation sequencing (NGS)</em> technologies (e.g.Â Roche 454 Life Sciences Pyrosequencing, Illumina, and lately Oxford Nanopore) as they are significantly cheaper and faster <span class="citation">(Metzker, 2010)</span>. By isolating and sequencing a relatively short part of a âfingerprintâ gene like the 16S ribosomal RNA (16S rRNA) gene, it is possible to identify the microorganisms present based on variable regions in the gene (Figure <a href="1-introduction.html#fig:16svariableregions">1.1</a>) within a day.</p>
<div class="figure" style="text-align: center"><span id="fig:16svariableregions"></span>
<img src="figure/16srRNA.png" alt="The secondary structure of the 16S rRNA gene of Escherichia coli with the variable regions V1-V9 indicated with bold text. The colors represent the separate regions of the gene that have been sequenced individually. Source: (Yarza et al, 2014)" width="450px" />
<p class="caption">
Figure 1.1: The secondary structure of the 16S rRNA gene of Escherichia coli with the variable regions V1-V9 indicated with bold text. The colors represent the separate regions of the gene that have been sequenced individually. Source: (Yarza et al, 2014)
</p>
</div>
<p></p>
<p>Due to its vital function in the ribosome for protein synthesis, the 16S rRNA gene is a highly conserved gene found in almost all prokaryotes <span class="citation">(Clarridge, 2004)</span>. Since the final gene product is RNA, mutations are fatal since the exact nucleotide sequence is essential to preserve its activity <span class="citation">(D. Qin, Abdi, &amp; Fredrick, 2007)</span>. Some regions of the gene are more important than others, however, which allows for evolutionary relationships to be drawn from the less preserved, variable regions <span class="citation">(Clarridge, 2004)</span>. In the 16S rRNA gene there are 9 of these regions, V1-V9, (Figure <a href="1-introduction.html#fig:16svariableregions">1.1</a>), which can then be used as phylogenetic markers <span class="citation">(Ashelford, Chuzhanova, Fry, Jones, &amp; Weightman, 2005; Woese &amp; Fox, 1977)</span>. By sequencing the DNA of one or more of these variable regions of the 16S rRNA gene using NGS, it is then possible to identify the majority of microorganisms present in a sample. This method is called 16S rRNA <em>Amplicon Sequencing</em> and is today widely used and the preferred choice to analyse microbial communities <span class="citation">(M. C. van Loosdrecht et al., 2016)</span>. The fundamental steps of <em>Amplicon Sequencing</em> (Figure <a href="1-introduction.html#fig:ampliconsequencing">1.2</a>) requires both laboratory practices with DNA extraction, PCR amplification of one or more of the variable regions, and subsequent sequencing of the DNA libraries, as well as bioinformatic processing of the obtained DNA reads.</p>
<!--
microbial dark matter [@Rinke2013]
-->
<div class="figure" style="text-align: center"><span id="fig:ampliconsequencing"></span>
<img src="figure/amplicon.png" alt="The fundamental steps in the analysis of microbial communities using 16S amplicon sequencing. Source: (Loosdrecht et al, 2016)" width="450px" />
<p class="caption">
Figure 1.2: The fundamental steps in the analysis of microbial communities using 16S amplicon sequencing. Source: (Loosdrecht et al, 2016)
</p>
</div>
<p>The latter step involves clustering of the reads by sequence identity into <em>Operational Taxonomic Units</em> (OTUs), which are then inferred taxonomic classification based on a suited reference database, of which the most commonly used today are SILVA <span class="citation">(Quast et al., 2013)</span>, RDP <span class="citation">(Cole et al., 2014)</span>, Greengenes <span class="citation">(McDonald et al., 2012)</span>, or specifically for AS, the MiDAS database <span class="citation">(S. J. McIlroy et al., 2015)</span>. The next step is data analysis, which highly depends on the purpose of the particular study.</p>
</div>
<div id="purpose" class="section level2">
<h2><span class="header-section-number">1.3</span> Purpose</h2>
<p>The rapid development of NGS technologies within the last decades has resulted in a growing need for sofisticated bioinformatic tools to analyse the large, complex and multivariate data sets obtained with these high-throughput technologies. The purpose of this study is to develop bioinformatic tools to ease the analysis of data obtained using NGS. Specifically, the usefulness of ordination for the analysis and comparison of the microbial communities of the activated sludge in Danish wastewater treatment plants will be investigated, and a handful of questions (listed in Aims) will be answered using primarily ordination.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="2-ordination.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
