<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>The Activated Sludge of Danish Wastewater Treatment Plants</title>
  <meta name="description" content="The Activated Sludge of Danish Wastewater Treatment Plants">
  <meta name="generator" content="bookdown 0.3.19 and GitBook 2.6.7">

  <meta property="og:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  
  
  

<meta name="author" content="Kasper Skytte Andersen">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="aims.html">
<link rel="next" href="4-results1.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-0.8/htmlwidgets.js"></script>
<link href="libs/plotlyjs-1.26.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.26.1/plotly-latest.min.js"></script>
<script src="libs/plotly-binding-4.6.0/plotly.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#the-history-of-wastewater-treatment-and-the-activated-sludge-process"><i class="fa fa-check"></i><b>1.1</b> The history of wastewater treatment and the activated sludge process</a></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#identification-of-microorganisms-in-activated-sludge"><i class="fa fa-check"></i><b>1.2</b> Identification of microorganisms in activated sludge</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#purpose"><i class="fa fa-check"></i><b>1.3</b> Purpose</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-ordination.html"><a href="2-ordination.html"><i class="fa fa-check"></i><b>2</b> Ordination in Microbial Ecology</a><ul>
<li class="chapter" data-level="2.1" data-path="2-ordination.html"><a href="2-ordination.html#exploratoryvsexplanatory"><i class="fa fa-check"></i><b>2.1</b> Exploratory vs. Explanatory</a></li>
<li class="chapter" data-level="2.2" data-path="2-ordination.html"><a href="2-ordination.html#nichedoublezero"><i class="fa fa-check"></i><b>2.2</b> Niche theory and the double-zero problem</a></li>
<li class="chapter" data-level="2.3" data-path="2-ordination.html"><a href="2-ordination.html#distance-based-ordination"><i class="fa fa-check"></i><b>2.3</b> Distance-based ordination</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-ordination.html"><a href="2-ordination.html#principal-coordinates-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Principal Coordinates Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-ordination.html"><a href="2-ordination.html#non-metric-multidimensional-scaling"><i class="fa fa-check"></i><b>2.3.2</b> non-Metric Multidimensional Scaling</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-ordination.html"><a href="2-ordination.html#metrics"><i class="fa fa-check"></i><b>2.3.3</b> Distance- and (dis)similarity measures</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-ordination.html"><a href="2-ordination.html#eigenanalysis-based-ordination"><i class="fa fa-check"></i><b>2.4</b> Eigenanalysis-based ordination</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-ordination.html"><a href="2-ordination.html#PCA"><i class="fa fa-check"></i><b>2.4.1</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-ordination.html"><a href="2-ordination.html#redundancy-analysis"><i class="fa fa-check"></i><b>2.4.2</b> Redundancy Analysis</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-ordination.html"><a href="2-ordination.html#CA"><i class="fa fa-check"></i><b>2.4.3</b> Correspondence Analysis and Canonical Correspondence Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims</a></li>
<li class="chapter" data-level="3" data-path="3-mm.html"><a href="3-mm.html"><i class="fa fa-check"></i><b>3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="3.1" data-path="3-mm.html"><a href="3-mm.html#sampling"><i class="fa fa-check"></i><b>3.1</b> Sampling</a></li>
<li class="chapter" data-level="3.2" data-path="3-mm.html"><a href="3-mm.html#dna-extraction"><i class="fa fa-check"></i><b>3.2</b> DNA extraction</a></li>
<li class="chapter" data-level="3.3" data-path="3-mm.html"><a href="3-mm.html#library-preparation-purification-and-pooling"><i class="fa fa-check"></i><b>3.3</b> Library preparation, purification and pooling</a></li>
<li class="chapter" data-level="3.4" data-path="3-mm.html"><a href="3-mm.html#dna-sequencing-and-bioinformatics"><i class="fa fa-check"></i><b>3.4</b> DNA sequencing and bioinformatics</a></li>
<li class="chapter" data-level="3.5" data-path="3-mm.html"><a href="3-mm.html#data-processing-and-analysis"><i class="fa fa-check"></i><b>3.5</b> Data processing and analysis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-results1.html"><a href="4-results1.html"><i class="fa fa-check"></i><b>4</b> Exploring the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="4.1" data-path="4-results1.html"><a href="4-results1.html#overview-of-the-differences"><i class="fa fa-check"></i><b>4.1</b> Overview of the differences</a></li>
<li class="chapter" data-level="4.2" data-path="4-results1.html"><a href="4-results1.html#wwtpcommunities"><i class="fa fa-check"></i><b>4.2</b> How does the microbial community composition describe the WWTPs?</a></li>
<li class="chapter" data-level="4.3" data-path="4-results1.html"><a href="4-results1.html#concluding-remarks"><i class="fa fa-check"></i><b>4.3</b> Concluding remarks</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-results2.html"><a href="5-results2.html"><i class="fa fa-check"></i><b>5</b> Explaining the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="5.1" data-path="5-results2.html"><a href="5-results2.html#the-influence-of-plant-design-on-the-microbial-communities"><i class="fa fa-check"></i><b>5.1</b> The influence of plant design on the microbial communities</a></li>
<li class="chapter" data-level="5.2" data-path="5-results2.html"><a href="5-results2.html#general-differences-related-to-sampling-time"><i class="fa fa-check"></i><b>5.2</b> General differences related to sampling time</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-discussion.html"><a href="6-discussion.html"><i class="fa fa-check"></i><b>6</b> General Discussion</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplementaryplots.html"><a href="A-supplementaryplots.html"><i class="fa fa-check"></i><b>A</b> Supplementary plots</a></li>
<li class="chapter" data-level="B" data-path="B-screeplots.html"><a href="B-screeplots.html"><i class="fa fa-check"></i><b>B</b> Scree plots</a><ul>
<li class="chapter" data-level="B.1" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpca"><i class="fa fa-check"></i><b>B.1</b> Scree plot of Figure @ref(fig:overviewPCA)</a></li>
<li class="chapter" data-level="B.2" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpcoabray"><i class="fa fa-check"></i><b>B.2</b> Scree plot of Figure @ref(fig:overviewPCoAbray)</a></li>
<li class="chapter" data-level="B.3" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewcca"><i class="fa fa-check"></i><b>B.3</b> Scree plot of Figure @ref(fig:overviewCCA)</a></li>
<li class="chapter" data-level="B.4" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigfactorccaa-d"><i class="fa fa-check"></i><b>B.4</b> Scree plot of Figure @ref(fig:factorCCA)(A-D)</a></li>
<li class="chapter" data-level="B.5" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigtimeccaab"><i class="fa fa-check"></i><b>B.5</b> Scree plot of Figure @ref(fig:timeCCA)(A+B)</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-WWTPcharacteristics.html"><a href="C-WWTPcharacteristics.html"><i class="fa fa-check"></i><b>C</b> Characteristics of the WWTPs</a></li>
<li class="chapter" data-level="D" data-path="D-references.html"><a href="D-references.html"><i class="fa fa-check"></i><b>D</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Activated Sludge of Danish Wastewater Treatment Plants</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mm" class="section level1">
<h1><span class="header-section-number">3</span> Materials and Methods</h1>
<p>The experimental procedures performed will only be covered briefly here. For detailed protocols refer to the 3 protocols “DNA extraction from activated sludge”, “MiSeq Sequencing of Amplicons” and “16S rRNA V1-3 Amplicon Preparation v1.2” at <a href="http://midasfieldguide.org/en/protocols/" class="uri">http://midasfieldguide.org/en/protocols/</a> or the supplementary data from <span class="citation">(S. J. McIlroy et al., 2015)</span>.</p>
<div id="sampling" class="section level2">
<h2><span class="header-section-number">3.1</span> Sampling</h2>
<p>The activated sludge (AS) from 32 Danish wastewater treatment plants (WWTPs) were sampled at a 1 meter depth below the surface. Some WWTPs were sampled 2 times a year (summer, winter), while others 4 times a year (summer, fall, winter, and spring), in the period 2006 to 2015. The samples were kept on ice as much as possible and stored at -80ºC.</p>
</div>
<div id="dna-extraction" class="section level2">
<h2><span class="header-section-number">3.2</span> DNA extraction</h2>
<p>The DNA was extracted based on the manufacturer’s instructions of the FastDNA™ 2 mL SPIN Kit for Soil (MP Biomedicals, USA), optimised for DNA extraction from activated sludge by using increased bead beating with 4x40s at 6 m/s.</p>
<p></p>
</div>
<div id="library-preparation-purification-and-pooling" class="section level2">
<h2><span class="header-section-number">3.3</span> Library preparation, purification and pooling</h2>
<p>Polymerase Chain Reaction (PCR) was used to amplify the V1-3 variable regions of the 16S rRNA gene using the barcoded forward (F) and reverse (R) primers listed below:</p>
<ul>
<li>27F: AGAGTTTGATCCTGGCTCAG</li>
<li>534R: ATTACCGCGGCTGCTGG</li>
</ul>
<p>PCR was performed with the thermo cycler settings: Initial denaturation at 95 °C for 2 min, 30 cycles of 95°C for 20s, 56°C for 30s, 72°C for 60s and final elongation at 72°C for 5 min. All PCR reactions were run in duplicates and pooled afterwards. The amplicon libraries were purified using the Agencourt® AMpure XP bead kit (Beckmann Coulter, USA) and the DNA concentration and quality was assured afterwards using a Quant-iT™ HS DNA Assay (Thermo Fisher Scientific) and Tapestation 2200 with D1K ScreenTapes (Agilent), respectively. Based on the measured library DNA concentrations the samples were pooled in equimolar concentrations before sequencing.</p>
</div>
<div id="dna-sequencing-and-bioinformatics" class="section level2">
<h2><span class="header-section-number">3.4</span> DNA sequencing and bioinformatics</h2>
<p>The libraries were then sequenced using a MiSeq (Illumina, USA) with a PhiX control library. The libraries were subsampled to 50000 raw reads, and low quality reads were removed using Trimmomatic v0.32. Using USEARCH9 <span class="citation">(Edgar, 2013)</span>, the sequence pairs were merged, PhiX sequences were filtered, unique sequences were identified, classified and clustered into Operational Taxonomic Units (OTUs, <span class="math inline">\(\geq\)</span> 97% sequence identity). The OTUs were then infered taxonomic identification based on the MiDAS taxonomic database <span class="citation">(S. J. McIlroy et al., 2015)</span>, which is a manually curated version of the SILVA database <span class="citation">(Quast et al., 2013)</span> suited for activated sludge microorganisms. The result is a sample-by-OTU table which contains the approximate read abundances of each OTU in each sample alongside their taxonomy.</p>
</div>
<div id="data-processing-and-analysis" class="section level2">
<h2><span class="header-section-number">3.5</span> Data processing and analysis</h2>
<p>The raw sample-by-OTU table was analysed using the R statistical language <span class="citation">(Ihaka &amp; Gentleman, 1996)</span> and the RStudio IDE (<a href="https://www.rstudio.org" class="uri">https://www.rstudio.org</a>). I wrote an R-function (500+ lines of code) based on the vegan and ggplot2 R-packages for ordination of the data called ord_mep() (ordination of microbial ecology profiles) specifically for the purpose of this report, available at <a href="https://github.com/KasperSkytte/ord_mep" class="uri">https://github.com/KasperSkytte/ord_mep</a>. The function is inspired by the function amp_ordinate() from the R-package “ampvis” by Mads Albertsen <span class="citation">(Albertsen, Karst, Ziegler, Kirkegaard, &amp; Nielsen, 2015)</span>, extended with support for all ordination methods mentioned in Chapter <a href="2-ordination.html#ordination">2</a>, data transformations, plotting options, and more.</p>
<p>The OTUs have been filtered so that only those with a read abundance larger than 0.1% (across <em>all</em> samples) in at least one sample are being analysed. If not otherwise noted, ordination of the data has been done by first applying the Hellinger transformation to account for incorrect similarities due to double-zeros and to also downweight low abundant species <span class="citation">(Buttigieg &amp; Ramette, 2014; P. Legendre &amp; Gallagher, 2001)</span>:</p>

<p><strong>The Hellinger transformation</strong></p>
<span class="math display">\[\begin{equation}
  D_{Hellinger}(x_1,x_2)=
  \sqrt{
  \sum_{j=1}^p
  \left(
  \sqrt{\frac{y_{1j}}{y_{1+}}}-
  \sqrt{\frac{y_{2j}}{y_{2+}}}
  \right)^2
  }
  \label{eq:hellinger}
\end{equation}\]</span>
<p>where <span class="math inline">\(Y=[y_{ij}]\)</span> is a species abundance table of the size <span class="math inline">\((n{\times}p)\)</span> with sites (rows) <span class="math inline">\(i=[1...n]\)</span>, species (columns) <span class="math inline">\(j=[1...p]\)</span> and row sums <span class="math inline">\(y_{i+}\)</span>. <span class="citation">(P. Legendre &amp; Gallagher, 2001)</span></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="aims.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-results1.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
