<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>The Activated Sludge of Danish Wastewater Treatment Plants</title>
  <meta name="description" content="The Activated Sludge of Danish Wastewater Treatment Plants">
  <meta name="generator" content="bookdown 0.3.19 and GitBook 2.6.7">

  <meta property="og:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  
  
  

<meta name="author" content="Kasper Skytte Andersen">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="5-results2.html">
<link rel="next" href="conclusion.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-0.8/htmlwidgets.js"></script>
<link href="libs/plotlyjs-1.26.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.26.1/plotly-latest.min.js"></script>
<script src="libs/plotly-binding-4.6.0/plotly.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#the-history-of-wastewater-treatment-and-the-activated-sludge-process"><i class="fa fa-check"></i><b>1.1</b> The history of wastewater treatment and the activated sludge process</a></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#identification-of-microorganisms-in-activated-sludge"><i class="fa fa-check"></i><b>1.2</b> Identification of microorganisms in activated sludge</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#purpose"><i class="fa fa-check"></i><b>1.3</b> Purpose</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-ordination.html"><a href="2-ordination.html"><i class="fa fa-check"></i><b>2</b> Ordination in Microbial Ecology</a><ul>
<li class="chapter" data-level="2.1" data-path="2-ordination.html"><a href="2-ordination.html#exploratoryvsexplanatory"><i class="fa fa-check"></i><b>2.1</b> Exploratory vs.Â Explanatory</a></li>
<li class="chapter" data-level="2.2" data-path="2-ordination.html"><a href="2-ordination.html#nichedoublezero"><i class="fa fa-check"></i><b>2.2</b> Niche theory and the double-zero problem</a></li>
<li class="chapter" data-level="2.3" data-path="2-ordination.html"><a href="2-ordination.html#distance-based-ordination"><i class="fa fa-check"></i><b>2.3</b> Distance-based ordination</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-ordination.html"><a href="2-ordination.html#principal-coordinates-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Principal Coordinates Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-ordination.html"><a href="2-ordination.html#non-metric-multidimensional-scaling"><i class="fa fa-check"></i><b>2.3.2</b> non-Metric Multidimensional Scaling</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-ordination.html"><a href="2-ordination.html#metrics"><i class="fa fa-check"></i><b>2.3.3</b> Distance- and (dis)similarity measures</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-ordination.html"><a href="2-ordination.html#eigenanalysis-based-ordination"><i class="fa fa-check"></i><b>2.4</b> Eigenanalysis-based ordination</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-ordination.html"><a href="2-ordination.html#PCA"><i class="fa fa-check"></i><b>2.4.1</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-ordination.html"><a href="2-ordination.html#redundancy-analysis"><i class="fa fa-check"></i><b>2.4.2</b> Redundancy Analysis</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-ordination.html"><a href="2-ordination.html#CA"><i class="fa fa-check"></i><b>2.4.3</b> Correspondence Analysis and Canonical Correspondence Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims</a></li>
<li class="chapter" data-level="3" data-path="3-mm.html"><a href="3-mm.html"><i class="fa fa-check"></i><b>3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="3.1" data-path="3-mm.html"><a href="3-mm.html#sampling"><i class="fa fa-check"></i><b>3.1</b> Sampling</a></li>
<li class="chapter" data-level="3.2" data-path="3-mm.html"><a href="3-mm.html#dna-extraction"><i class="fa fa-check"></i><b>3.2</b> DNA extraction</a></li>
<li class="chapter" data-level="3.3" data-path="3-mm.html"><a href="3-mm.html#library-preparation-purification-and-pooling"><i class="fa fa-check"></i><b>3.3</b> Library preparation, purification and pooling</a></li>
<li class="chapter" data-level="3.4" data-path="3-mm.html"><a href="3-mm.html#dna-sequencing-and-bioinformatics"><i class="fa fa-check"></i><b>3.4</b> DNA sequencing and bioinformatics</a></li>
<li class="chapter" data-level="3.5" data-path="3-mm.html"><a href="3-mm.html#data-processing-and-analysis"><i class="fa fa-check"></i><b>3.5</b> Data processing and analysis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-results1.html"><a href="4-results1.html"><i class="fa fa-check"></i><b>4</b> Exploring the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="4.1" data-path="4-results1.html"><a href="4-results1.html#overview-of-the-differences"><i class="fa fa-check"></i><b>4.1</b> Overview of the differences</a></li>
<li class="chapter" data-level="4.2" data-path="4-results1.html"><a href="4-results1.html#wwtpcommunities"><i class="fa fa-check"></i><b>4.2</b> How does the microbial community composition describe the WWTPs?</a></li>
<li class="chapter" data-level="4.3" data-path="4-results1.html"><a href="4-results1.html#concluding-remarks"><i class="fa fa-check"></i><b>4.3</b> Concluding remarks</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-results2.html"><a href="5-results2.html"><i class="fa fa-check"></i><b>5</b> Explaining the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="5.1" data-path="5-results2.html"><a href="5-results2.html#the-influence-of-plant-design-on-the-microbial-communities"><i class="fa fa-check"></i><b>5.1</b> The influence of plant design on the microbial communities</a></li>
<li class="chapter" data-level="5.2" data-path="5-results2.html"><a href="5-results2.html#general-differences-related-to-sampling-time"><i class="fa fa-check"></i><b>5.2</b> General differences related to sampling time</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-discussion.html"><a href="6-discussion.html"><i class="fa fa-check"></i><b>6</b> General Discussion</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplementaryplots.html"><a href="A-supplementaryplots.html"><i class="fa fa-check"></i><b>A</b> Supplementary plots</a></li>
<li class="chapter" data-level="B" data-path="B-screeplots.html"><a href="B-screeplots.html"><i class="fa fa-check"></i><b>B</b> Scree plots</a><ul>
<li class="chapter" data-level="B.1" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpca"><i class="fa fa-check"></i><b>B.1</b> Scree plot of Figure @ref(fig:overviewPCA)</a></li>
<li class="chapter" data-level="B.2" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpcoabray"><i class="fa fa-check"></i><b>B.2</b> Scree plot of Figure @ref(fig:overviewPCoAbray)</a></li>
<li class="chapter" data-level="B.3" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewcca"><i class="fa fa-check"></i><b>B.3</b> Scree plot of Figure @ref(fig:overviewCCA)</a></li>
<li class="chapter" data-level="B.4" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigfactorccaa-d"><i class="fa fa-check"></i><b>B.4</b> Scree plot of Figure @ref(fig:factorCCA)(A-D)</a></li>
<li class="chapter" data-level="B.5" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigtimeccaab"><i class="fa fa-check"></i><b>B.5</b> Scree plot of Figure @ref(fig:timeCCA)(A+B)</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-WWTPcharacteristics.html"><a href="C-WWTPcharacteristics.html"><i class="fa fa-check"></i><b>C</b> Characteristics of the WWTPs</a></li>
<li class="chapter" data-level="D" data-path="D-references.html"><a href="D-references.html"><i class="fa fa-check"></i><b>D</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Activated Sludge of Danish Wastewater Treatment Plants</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussion" class="section level1">
<h1><span class="header-section-number">6</span> General Discussion</h1>
<p>When analysing microbial communities using 16S rRNA Amplicon Sequencing, it is important to note that the obtained OTU abundances are nothing more than relative measures of the <em>read abundances</em> of the extracted DNA sequences in the samples relative to the total amount of reads. They do not exactly represent the amount of cells present of a particular microorganism due to various biases. These biases are introduced by for example multiple gene copy numbers (GCN) within each cell, primer specificity, and DNA extraction <span class="citation">(M. C. van Loosdrecht et al., 2016)</span>. A study in 2004 found that the 76 bacterial genomes sequenced contained between 1-15 operon copies and up to 7 are commonly found. Only 40% of the genomes had 1-2 <span class="citation">(Acinas, Marcelino, Klepac-Ceraj, &amp; Polz, 2004)</span>. This introduces a significant bias which can be further enhanced by multiple genome copies. Additionally, the primers used to target the 16S rRNA gene are designed based on a consensus sequence matching as many bacteria as possible, and therefore whole taxonomic groups can be overlooked. To provide a more complete picture of the microbial communities, perhaps all 9 variable regions of the 16S rRNA should be sequenced, or alternatively sequencing full-length ribosomal small subunits (SSUs) using the Oxford Nanopore instead <span class="citation">(S. M. Karst et al., 2016)</span>. Together, these biases may have had the consequence that the true ecological differences between the WWTPs are not exactly as presented here. Because the abundances do not seem to have a large influence on the differences when comparing the results of PCA (Figure <a href="4-results1.html#fig:overviewPCA">4.1</a>) with PCoA using the Bray-Curtis dissimilarity measure (Figure <a href="4-results1.html#fig:overviewPCoAbray">4.2</a>), biases in abundances are considered unimportant, however. If neccessary, tools exist to correct the abundances based on known GCNs of specific bacteria <span class="citation">(Angly et al., 2014)</span>.</p>
<p>The fact that the WWTPs have many OTUs in common is supported by the work of <span class="citation">A. M. Saunders, Albertsen, Vollertsen, &amp; Nielsen (2016)</span>, where they found that 13 Danish WWTPs (of which many are the same as those investigated in this report) contained 63 genera making up 68% of the total reads using 16S rRNA amplicon sequencing. In one of the WWTPs, they further related these genera to what was present in the influent wastewater, concluding that 10% of the total reads were due to immigration from the influent wastewater, and that these OTUs participated little to the wastewater treatment based on their growth rate in the AS. This helps elucidate the influence of the bacteria present in the influent wastewater on the microbial community of the activated sludge. The exact influence of the influent wastewater to the WWTPs sampled here have not been investigated and is a subject for further study.</p>
<p>There have not previously been performed many studies of this kind using different ordination methods to describe so many samples at once, and in particular from the AS of different WWTPs. The largest similar study to date seems to be of GenBank records of 202 globally distributed environmental samples from different soil and water environments <span class="citation">(C. A. Lozupone, Hamady, Kelley, &amp; Knight, 2007)</span>. The authors used PCoA of UniFrac distances, which incorporates phylogenetic distances <span class="citation">(Lozupone &amp; Knight, 2005)</span>, to identify clusters of samples based on known physical environmental factors. They found that salinity was a major environmental determinant of the microbial community compositions, more impactful than other physical factors like temperature or pH. This supports that the halophilic genera observed in the Ribe WWTP (Figure <a href="4-results1.html#fig:speciesCCA">4.6</a>) indeed could be due to high salinity in the wastewater. In another study of 14 geographically separated WWTPs in China and US <span class="citation">(Zhang, Shao, &amp; Ye, 2012)</span>, they performed both Cluster Analysis using the Bray-Curtis dissimilarity measure aswell as PCoA of weighted UniFrac distances on data obtained using Roche 454 pyrosequencing of 16S rRNA amplicons. They were able to roughly identify 3 clusters of samples from the WWTPs, however only 15 samples were analysed. Interestingly, like <span class="citation">A. M. Saunders et al. (2016)</span>, they also found that the majority (70 to be exact) of the 744 identified genera present in all 15 samples made up a large amount of the OTU reads (63.7%), which supports that a few genera can be dominant in most WWTPs, as also observed in this study (5 out of all 364 identified genera made up ~20% of the reads). Furthermore, <span class="citation">Zhang et al. (2012)</span> reported that some of the most abundant genera identified in the 14 WWTPs were <em>Zoogloea</em>, <em>Trichococcus</em>, <em>Prosthecobacter</em>, and <em>Dechloromonas</em> (in decreasing order of abundance), where <em>Zoogloea</em> and <em>Prosthecobacter</em> are not observed among the 40 most abundant genera in this study (Figure <a href="4-results1.html#fig:overviewheatmap">4.4</a>). This is possibly due to a different climate or other factors, however.</p>
<p>The fact that the WWTPs seem similar and have many OTUs in common is further confirmed by other ordination methods than the ones presented in this report. There are numerous combinations of data transformation and filtering, ordination methods, and distance measures which can be used to analyse the microbial communities of the WWTPs, however only a handful of the most informative have been shown in this report. Several other methods showed somewhat the same patterns, and in general they all indicated that the WWTPs are similar with shared OTUs (a few additional plots can be found in Appendix <a href="A-supplementaryplots.html#supplementaryplots">A</a>). Categorising the microbial communities of WWTPs using ordination was initially inspired by the clustering of human gut microbiomes into so-called <em>Enterotypes</em> <span class="citation">(Arumugam et al., 2011)</span>. Here, the authors particularly used one measure only (Jensen-Shannon Divergence) to cluster the microbiomes with little reasoning noted, and the practice of <em>Enterotyping</em> has received criticism <span class="citation">(Knights et al., 2014; Koren et al., 2013)</span>, because the clustering of microbial communities highly depends on the chosen distance measure. This is also evident in the ordination analyses in this report, where different measures revealed different results from the exact same data, which therefore highlights the importance of knowing what kind of information the particular measure is able to reveal in the data and use this as an advantage. CCA were able to reveal unique OTUs in the WWTPs, while PCA and PCoA instead highlighted differences based on OTUs generally with a high read abundance.</p>
<p>The eigenvalue percentages of the axes plotted are generally low in the ordination plots, especially in CCA. This is believed to be due to the large amount of similar samples being analysed at the same time. When analysing fewer samples from only one year, 2013 (Figure <a href="A-supplementaryplots.html#fig:CCA2013">A.7</a> in Appendix <a href="A-supplementaryplots.html#supplementaryplots">A</a>), the overall patterns are very similar to the patterns observed when all 622 samples are analysed at once (Figure <a href="4-results1.html#fig:overviewCCA">4.3</a>). Again, the microbial communities of the Ribe and Esbjerg E+W WWTPs seem to be different from the rest of the WWTPs, but the eigenvalue percentages of the axes are significantly higher (<em>all samples</em>: 5.8% and 3.7% - <em>2013 samples</em>: 13.9% and 9.5%). This further confirms that there are many similar OTUs between the samples, or else the eigenvalues would not be significantly lower with more samples analysed. In perspective, in the study mentioned earlier of 202 environmental samples <span class="citation">(C. A. Lozupone et al., 2007)</span>, the axis percentages of their PCoA analyses were similarly low (between 3.1%-5.5%), which indicates that this is not unusual when analysing this many samples at once.</p>
<p>As a last note, it would arguably make sense to filter the OTUs that are only present in one sample to provide a completely representative picture of the microbial communities of each WWTP, as these OTUs are possibly not part of the functional âcoreâ microbial community of the particular WWTP. As mentioned, there are many of these OTUs which are only present in one sample, and filtering them may have revealed a more ecologically meaningful representation of the WWTPs with respect to their performance.</p>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-results2.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
