<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>The Activated Sludge of Danish Wastewater Treatment Plants</title>
  <meta name="description" content="The Activated Sludge of Danish Wastewater Treatment Plants">
  <meta name="generator" content="bookdown 0.3.19 and GitBook 2.6.7">

  <meta property="og:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="The Activated Sludge of Danish Wastewater Treatment Plants" />
  
  
  

<meta name="author" content="Kasper Skytte Andersen">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="C-WWTPcharacteristics.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-0.8/htmlwidgets.js"></script>
<link href="libs/plotlyjs-1.26.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.26.1/plotly-latest.min.js"></script>
<script src="libs/plotly-binding-4.6.0/plotly.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#the-history-of-wastewater-treatment-and-the-activated-sludge-process"><i class="fa fa-check"></i><b>1.1</b> The history of wastewater treatment and the activated sludge process</a></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#identification-of-microorganisms-in-activated-sludge"><i class="fa fa-check"></i><b>1.2</b> Identification of microorganisms in activated sludge</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#purpose"><i class="fa fa-check"></i><b>1.3</b> Purpose</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-ordination.html"><a href="2-ordination.html"><i class="fa fa-check"></i><b>2</b> Ordination in Microbial Ecology</a><ul>
<li class="chapter" data-level="2.1" data-path="2-ordination.html"><a href="2-ordination.html#exploratoryvsexplanatory"><i class="fa fa-check"></i><b>2.1</b> Exploratory vs. Explanatory</a></li>
<li class="chapter" data-level="2.2" data-path="2-ordination.html"><a href="2-ordination.html#nichedoublezero"><i class="fa fa-check"></i><b>2.2</b> Niche theory and the double-zero problem</a></li>
<li class="chapter" data-level="2.3" data-path="2-ordination.html"><a href="2-ordination.html#distance-based-ordination"><i class="fa fa-check"></i><b>2.3</b> Distance-based ordination</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-ordination.html"><a href="2-ordination.html#principal-coordinates-analysis"><i class="fa fa-check"></i><b>2.3.1</b> Principal Coordinates Analysis</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-ordination.html"><a href="2-ordination.html#non-metric-multidimensional-scaling"><i class="fa fa-check"></i><b>2.3.2</b> non-Metric Multidimensional Scaling</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-ordination.html"><a href="2-ordination.html#metrics"><i class="fa fa-check"></i><b>2.3.3</b> Distance- and (dis)similarity measures</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-ordination.html"><a href="2-ordination.html#eigenanalysis-based-ordination"><i class="fa fa-check"></i><b>2.4</b> Eigenanalysis-based ordination</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-ordination.html"><a href="2-ordination.html#PCA"><i class="fa fa-check"></i><b>2.4.1</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-ordination.html"><a href="2-ordination.html#redundancy-analysis"><i class="fa fa-check"></i><b>2.4.2</b> Redundancy Analysis</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-ordination.html"><a href="2-ordination.html#CA"><i class="fa fa-check"></i><b>2.4.3</b> Correspondence Analysis and Canonical Correspondence Analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims</a></li>
<li class="chapter" data-level="3" data-path="3-mm.html"><a href="3-mm.html"><i class="fa fa-check"></i><b>3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="3.1" data-path="3-mm.html"><a href="3-mm.html#sampling"><i class="fa fa-check"></i><b>3.1</b> Sampling</a></li>
<li class="chapter" data-level="3.2" data-path="3-mm.html"><a href="3-mm.html#dna-extraction"><i class="fa fa-check"></i><b>3.2</b> DNA extraction</a></li>
<li class="chapter" data-level="3.3" data-path="3-mm.html"><a href="3-mm.html#library-preparation-purification-and-pooling"><i class="fa fa-check"></i><b>3.3</b> Library preparation, purification and pooling</a></li>
<li class="chapter" data-level="3.4" data-path="3-mm.html"><a href="3-mm.html#dna-sequencing-and-bioinformatics"><i class="fa fa-check"></i><b>3.4</b> DNA sequencing and bioinformatics</a></li>
<li class="chapter" data-level="3.5" data-path="3-mm.html"><a href="3-mm.html#data-processing-and-analysis"><i class="fa fa-check"></i><b>3.5</b> Data processing and analysis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-results1.html"><a href="4-results1.html"><i class="fa fa-check"></i><b>4</b> Exploring the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="4.1" data-path="4-results1.html"><a href="4-results1.html#overview-of-the-differences"><i class="fa fa-check"></i><b>4.1</b> Overview of the differences</a></li>
<li class="chapter" data-level="4.2" data-path="4-results1.html"><a href="4-results1.html#wwtpcommunities"><i class="fa fa-check"></i><b>4.2</b> How does the microbial community composition describe the WWTPs?</a></li>
<li class="chapter" data-level="4.3" data-path="4-results1.html"><a href="4-results1.html#concluding-remarks"><i class="fa fa-check"></i><b>4.3</b> Concluding remarks</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-results2.html"><a href="5-results2.html"><i class="fa fa-check"></i><b>5</b> Explaining the Microbial Communities of the WWTPs</a><ul>
<li class="chapter" data-level="5.1" data-path="5-results2.html"><a href="5-results2.html#the-influence-of-plant-design-on-the-microbial-communities"><i class="fa fa-check"></i><b>5.1</b> The influence of plant design on the microbial communities</a></li>
<li class="chapter" data-level="5.2" data-path="5-results2.html"><a href="5-results2.html#general-differences-related-to-sampling-time"><i class="fa fa-check"></i><b>5.2</b> General differences related to sampling time</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-discussion.html"><a href="6-discussion.html"><i class="fa fa-check"></i><b>6</b> General Discussion</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supplementaryplots.html"><a href="A-supplementaryplots.html"><i class="fa fa-check"></i><b>A</b> Supplementary plots</a></li>
<li class="chapter" data-level="B" data-path="B-screeplots.html"><a href="B-screeplots.html"><i class="fa fa-check"></i><b>B</b> Scree plots</a><ul>
<li class="chapter" data-level="B.1" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpca"><i class="fa fa-check"></i><b>B.1</b> Scree plot of Figure @ref(fig:overviewPCA)</a></li>
<li class="chapter" data-level="B.2" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewpcoabray"><i class="fa fa-check"></i><b>B.2</b> Scree plot of Figure @ref(fig:overviewPCoAbray)</a></li>
<li class="chapter" data-level="B.3" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigoverviewcca"><i class="fa fa-check"></i><b>B.3</b> Scree plot of Figure @ref(fig:overviewCCA)</a></li>
<li class="chapter" data-level="B.4" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigfactorccaa-d"><i class="fa fa-check"></i><b>B.4</b> Scree plot of Figure @ref(fig:factorCCA)(A-D)</a></li>
<li class="chapter" data-level="B.5" data-path="B-screeplots.html"><a href="B-screeplots.html#scree-plot-of-figure-reffigtimeccaab"><i class="fa fa-check"></i><b>B.5</b> Scree plot of Figure @ref(fig:timeCCA)(A+B)</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-WWTPcharacteristics.html"><a href="C-WWTPcharacteristics.html"><i class="fa fa-check"></i><b>C</b> Characteristics of the WWTPs</a></li>
<li class="chapter" data-level="D" data-path="D-references.html"><a href="D-references.html"><i class="fa fa-check"></i><b>D</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Activated Sludge of Danish Wastewater Treatment Plants</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1">
<h1><span class="header-section-number">D</span> References</h1>
<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->
<p>  </p>
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references">
<div>
<p>Acinas, S. G., Marcelino, L. A., Klepac-Ceraj, V., &amp; Polz, M. F. (2004). Divergence and Redundancy of 16S rRNA Sequences in Genomes with Multiple rrn Operons. <em>Journal of Bacteriology</em>, <em>186</em>(9), 2629–2635. <a href="http://doi.org/10.1128/JB.186.9.2629-2635.2004" class="uri">http://doi.org/10.1128/JB.186.9.2629-2635.2004</a></p>
</div>
<div>
<p>Albertsen, M., Karst, S. M., Ziegler, A. S., Kirkegaard, R. H., &amp; Nielsen, P. H. (2015). Back to basics - The influence of DNA extraction and primer choice on phylogenetic analysis of activated sludge communities. <em>PLoS ONE</em>, <em>10</em>(7), 1–15. <a href="http://doi.org/10.1371/journal.pone.0132783" class="uri">http://doi.org/10.1371/journal.pone.0132783</a></p>
</div>
<div>
<p>Amann, R. I., Binder, B. J., Olson, R. J., Chisholm, S. W., Devereux, R., &amp; Stahl, D. A. (1990). Combination of 16S rRNA-targeted oligonucleotide probes with flow cytometry for analyzing mixed microbial populations. <em>Applied and Environmental Microbiology</em>, <em>56</em>(6), 1919–25. <a href="http://doi.org/0099-2240/90/061919-07" class="uri">http://doi.org/0099-2240/90/061919-07</a></p>
</div>
<div>
<p>Angly, F. E., Dennis, P. G., Skarshewski, A., Vanwonterghem, I., Hugenholtz, P., &amp; Tyson, G. W. (2014). CopyRighter: a rapid tool for improving the accuracy of microbial community profiles through lineage-specific gene copy number correction. <em>Microbiome</em>, <em>2</em>(1), 11. <a href="http://doi.org/10.1186/2049-2618-2-11" class="uri">http://doi.org/10.1186/2049-2618-2-11</a></p>
</div>
<div>
<p>Ardern, E., &amp; Lockett, W. T. (1914). Experiments on the Oxidation of Sewage without the Aid of Filters. <em>Journal of Society of Chemical Industry</em>, <em>33</em>(10), 523–539. <a href="http://doi.org/10.1002/jctb.5000331005" class="uri">http://doi.org/10.1002/jctb.5000331005</a></p>
</div>
<div>
<p>Arumugam, M., Raes, J., Pelletier, E., Le Paslier, D., Yamada, T., Mende, D. R., … Bork, P. (2011). Enterotypes of the human gut microbiome. <em>Nature</em>, <em>473</em>(7346), 174–180. <a href="http://doi.org/10.1038/nature09944" class="uri">http://doi.org/10.1038/nature09944</a></p>
</div>
<div>
<p>Ashelford, K. E., Chuzhanova, N. A., Fry, J. C., Jones, A. J., &amp; Weightman, A. J. (2005). At Least 1 in 20 16S rRNA Sequence Records Currently Held in Public Repositories Is Estimated To Contain Substantial Anomalies. <em>Applied and Environmental Microbiology</em>, <em>71</em>(12), 7724–7736. <a href="http://doi.org/10.1128/AEM.71.12.7724-7736.2005" class="uri">http://doi.org/10.1128/AEM.71.12.7724-7736.2005</a></p>
</div>
<div>
<p>Bae, H. S., Moe, W. M., Yan, J., Tiago, I., Costa, M. S. da, &amp; Rainey, F. A. (2006). Brooklawnia cerclae gen. nov., sp. nov., a propionate-forming bacterium isolated from chlorosolvent-contaminated groundwater. <em>International Journal of Systematic and Evolutionary Microbiology</em>, <em>56</em>(8), 1977–1983. <a href="http://doi.org/10.1099/ijs.0.64317-0" class="uri">http://doi.org/10.1099/ijs.0.64317-0</a></p>
</div>
<div>
<p>Braak, C. J. F. ter, &amp; Prentice, I. C. (1988). A Theory of Gradient Analysis. <em>Advances in Ecological Research</em>, <em>18</em>(C), 271–317. <a href="http://doi.org/10.1016/S0065-2504(08)60183-X" class="uri">http://doi.org/10.1016/S0065-2504(08)60183-X</a></p>
</div>
<div>
<p>Bray, J. R., &amp; Curtis, J. T. (1957). An Ordination of the Upland Forest Communities of Southern Wisconsin. <em>Ecological Monographs</em>, <em>27</em>(4), 325–349. <a href="http://doi.org/10.2307/1942268" class="uri">http://doi.org/10.2307/1942268</a></p>
</div>
<div>
<p>Buttigieg, P. L., &amp; Ramette, A. (2014). A guide to statistical analysis in microbial ecology: A community-focused, living review of multivariate data analyses. <em>FEMS Microbiology Ecology</em>, <em>90</em>(3), 543–550. <a href="http://doi.org/10.1111/1574-6941.12437" class="uri">http://doi.org/10.1111/1574-6941.12437</a></p>
</div>
<div>
<p>Clark, H. W., &amp; Adams, G. O. (1914). Sewage treatment by aeration and contact in tanks containing layers of slate. <em>Record</em>, <em>69</em>, 158–159.</p>
</div>
<div>
<p>Clarridge, J. E. (2004). Impact of 16S rRNA gene sequence analysis for identification of bacteria on clinical microbiology and infectious diseases. <em>Clinical Microbiology Reviews</em>, <em>17</em>(4), 840–62, tableof contents. <a href="http://doi.org/10.1128/CMR.17.4.840-862.2004" class="uri">http://doi.org/10.1128/CMR.17.4.840-862.2004</a></p>
</div>
<div>
<p>Cole, J. R., Wang, Q., Fish, J. A., Chai, B., McGarrell, D. M., Sun, Y., … Tiedje, J. M. (2014). Ribosomal Database Project: data and tools for high throughput rRNA analysis. <em>Nucleic Acids Research</em>, <em>42</em>(D1), D633–D642. <a href="http://doi.org/10.1093/nar/gkt1244" class="uri">http://doi.org/10.1093/nar/gkt1244</a></p>
</div>
<div>
<p>Dahle, H., &amp; Birkeland, N. K. (2006). Thermovirga lienii gen. nov., sp. nov., a novel moderately thermophilic, anaerobic, amino-acid-degrading bacterium isolated from a North Sea oil well. <em>International Journal of Systematic and Evolutionary Microbiology</em>, <em>56</em>(7), 1539–1545. <a href="http://doi.org/10.1099/ijs.0.63894-0" class="uri">http://doi.org/10.1099/ijs.0.63894-0</a></p>
</div>
<div>
<p>DeLong, E., Wickham, G., &amp; Pace, N. (1989). Phylogenetic stains: ribosomal RNA-based probes for the identification of single cells. <em>Science</em>, <em>243</em>(4896), 1360–1363. <a href="http://doi.org/10.1126/science.2466341" class="uri">http://doi.org/10.1126/science.2466341</a></p>
</div>
<div>
<p>Edgar, R. C. (2013). UPARSE: highly accurate OTU sequences from microbial amplicon reads. <em>Nature Methods</em>, <em>10</em>(10), 996–8. <a href="http://doi.org/10.1038/nmeth.2604" class="uri">http://doi.org/10.1038/nmeth.2604</a></p>
</div>
<div>
<p>Eikelboom, D. H. (1975). Filamentous organisms observed in activated sludge. <em>Water Research</em>, <em>9</em>(4), 365–388. <a href="http://doi.org/http://dx.doi.org/10.1016/0043-1354(75)90182-7" class="uri">http://doi.org/http://dx.doi.org/10.1016/0043-1354(75)90182-7</a></p>
</div>
<div>
<p>Fahrbach, M., Kuever, J., Meinke, R., Kämpfer, P., &amp; Hollender, J. (2006). Denitratisoma oestradiolicum gen. nov., sp. nov., a 17beta-oestradiol-degrading, denitrifying betaproteobacterium. <em>International Journal of Systematic and Evolutionary Microbiology</em>. <a href="http://doi.org/10.1099/ijs.0.63672-0" class="uri">http://doi.org/10.1099/ijs.0.63672-0</a></p>
</div>
<div>
<p>Fudou, R., Jojima, Y., Iizuka, T., &amp; Yamanaka, S. (2002). Haliangium ochraceum gen. nov., sp. nov. and Haliangium tepidum sp. nov.: novel moderately halophilic myxobacteria isolated from coastal saline environments. <em>The Journal of General and Applied Microbiology</em>, <em>48</em>(2), 109–116. <a href="http://doi.org/10.2323/jgam.48.109" class="uri">http://doi.org/10.2323/jgam.48.109</a></p>
</div>
<div>
<p>Fuller, G. W. (1915). RECENT DEVELOPMENTS IN SEWAGE DISPOSAL. <em>The Public Health Journal</em>, <em>6</em>(3), 103–106. Retrieved from <a href="http://www.jstor.org/stable/41996799" class="uri">http://www.jstor.org/stable/41996799</a></p>
</div>
<div>
<p>Goodall, D. (1953). Objective methods for the classification of vegetation. I. The use of positive interspecific correlation. <em>Australian Journal of Botany</em>, <em>1</em>(1), 39–63.</p>
</div>
<div>
<p>Hamlin, C. (1988). William Dibdin and the Idea of Biological Sewage Treatment. <em>Technology and Culture</em>, <em>29</em>(2), 189–218. <a href="http://doi.org/10.2307/3105523" class="uri">http://doi.org/10.2307/3105523</a></p>
</div>
<div>
<p>Hill, M. O. (1973). Reciprocal Averaging: An Eigenvector Method of Ordination. <em>The Journal of Ecology</em>, <em>61</em>(1), 237. <a href="http://doi.org/10.2307/2258931" class="uri">http://doi.org/10.2307/2258931</a></p>
</div>
<div>
<p>Hotelling, H. (1933). Analysis of a complex of statistical variables into principal components. <em>Journal of Educational Psychology</em>, <em>24</em>(6), 417–441. <a href="http://doi.org/10.1037/h0071325" class="uri">http://doi.org/10.1037/h0071325</a></p>
</div>
<div>
<p>Hutchinson, G. E. (1957). Concluding remarks. <em>Cold Spring Harbor Symposia on Quantitative Biology</em>, <em>22</em>, 415–427. <a href="http://doi.org/10.1101/SQB.1957.022.01.039" class="uri">http://doi.org/10.1101/SQB.1957.022.01.039</a></p>
</div>
<div>
<p>Ibarbalz, F. M., Figuerola, E. L., &amp; Erijman, L. (2013). Industrial activated sludge exhibit unique bacterial community composition at high taxonomic ranks. <em>Water Research</em>, <em>47</em>(11), 3854–3864. <a href="http://doi.org/10.1016/j.watres.2013.04.010" class="uri">http://doi.org/10.1016/j.watres.2013.04.010</a></p>
</div>
<div>
<p>Ihaka, R., &amp; Gentleman, R. (1996). R: A Language for Data Analysis and Graphics. <em>Journal of Computational and Graphical Statistics</em>, <em>5</em>(3), 299–314. <a href="http://doi.org/10.1080/10618600.1996.10474713" class="uri">http://doi.org/10.1080/10618600.1996.10474713</a></p>
</div>
<div>
<p>J Farrell, &amp; A Rose. (1967). Temperature Effects on Microorganisms. <em>Annual Review of Microbiology</em>, <em>21</em>(1), 101–120. <a href="http://doi.org/10.1146/annurev.mi.21.100167.000533" class="uri">http://doi.org/10.1146/annurev.mi.21.100167.000533</a></p>
</div>
<div>
<p>Karst, S. M., Dueholm, M. S., McIlroy, S. J., Kirkegaard, R. H., Nielsen, P. H., &amp; Albertsen, M. (2016). Thousands of primer-free, high-quality, full-length SSU rRNA sequences from all domains of life. <em>BioRxiv</em>. <a href="http://doi.org/10.1101/070771" class="uri">http://doi.org/10.1101/070771</a></p>
</div>
<div>
<p>Knights, D., Ward, T. L., McKinlay, C. E., Miller, H., Gonzalez, A., McDonald, D., &amp; Knight, R. (2014). Rethinking “Enterotypes”. <em>Cell Host &amp; Microbe</em>, <em>16</em>(4), 433–437. <a href="http://doi.org/10.1016/j.chom.2014.09.013" class="uri">http://doi.org/10.1016/j.chom.2014.09.013</a></p>
</div>
<div>
<p>Koren, O., Knights, D., Gonzalez, A., Waldron, L., Segata, N., Knight, R., … Ley, R. E. (2013). A Guide to Enterotypes across the Human Body: Meta-Analysis of Microbial Community Structures in Human Microbiome Datasets. <em>PLoS Computational Biology</em>, <em>9</em>(1). <a href="http://doi.org/10.1371/journal.pcbi.1002863" class="uri">http://doi.org/10.1371/journal.pcbi.1002863</a></p>
</div>
<div>
<p>Kristiansen, R., Nguyen, H. T. T., Saunders, A. M., Nielsen, J. L., Wimmer, R., Le, V. Q., … Nielsen, P. H. (2013). A metabolic model for members of the genus Tetrasphaera involved in enhanced biological phosphorus removal. <em>The ISME Journal</em>, <em>7</em>(3), 543–554. <a href="http://doi.org/10.1038/ismej.2012.136" class="uri">http://doi.org/10.1038/ismej.2012.136</a></p>
</div>
<div>
<p>Kruskal, J. B. (1964). Multidimensional scaling by optimizing goodness of fit to a nonmetric hypothesis. <em>Psychometrika</em>, <em>29</em>(1), 1–27. <a href="http://doi.org/10.1007/BF02289565" class="uri">http://doi.org/10.1007/BF02289565</a></p>
</div>
<div>
<p>Kuczynski, J., Liu, Z., Lozupone, C., McDonald, D., Fierer, N., &amp; Knight, R. (2010). Microbial community resemblance methods differ in their ability to detect biologically relevant patterns. <em>Nature Methods</em>, <em>7</em>(10), 813–819. <a href="http://doi.org/10.1038/nmeth.1499" class="uri">http://doi.org/10.1038/nmeth.1499</a></p>
</div>
<div>
<p>Legendre, P., &amp; Gallagher, E. (2001). Ecologically meaningful transformations for ordination of species data. <em>Oecologia</em>, <em>129</em>(2), 271–280. <a href="http://doi.org/10.1007/s004420100716" class="uri">http://doi.org/10.1007/s004420100716</a></p>
</div>
<div>
<p>Legendre, P., &amp; Legendre, L. (2012). <em>Numerical Ecology</em>. Elsevier Science. Retrieved from <a href="https://books.google.dk/books?id=6ZBOA-iDviQC" class="uri">https://books.google.dk/books?id=6ZBOA-iDviQC</a></p>
</div>
<div>
<p>Loosdrecht, M. C. van, Nielsen, P., Lopez-Vazquez, C., &amp; Brdjanovic, D. (2016). Experimental procedures in wastewater treatment. <em>Experimental Procedures in Wastewater Treatment</em>. Retrieved from <a href="http://www.iwapublishing.com/books/9781780404745/experimental-methods-wastewater-treatment" class="uri">http://www.iwapublishing.com/books/9781780404745/experimental-methods-wastewater-treatment</a></p>
</div>
<div>
<p>Lozupone, C. A., Hamady, M., Kelley, S. T., &amp; Knight, R. (2007). Quantitative and qualitative ?? diversity measures lead to different insights into factors that structure microbial communities. <em>Applied and Environmental Microbiology</em>, <em>73</em>(5), 1576–1585. <a href="http://doi.org/10.1128/AEM.01996-06" class="uri">http://doi.org/10.1128/AEM.01996-06</a></p>
</div>
<div>
<p>Lozupone, C., &amp; Knight, R. (2005). UniFrac : a New Phylogenetic Method for Comparing Microbial Communities UniFrac : a New Phylogenetic Method for Comparing Microbial Communities. <em>Applied and Environmental Microbiology</em>, <em>71</em>(12), 8228–8235. <a href="http://doi.org/10.1128/AEM.71.12.8228" class="uri">http://doi.org/10.1128/AEM.71.12.8228</a></p>
</div>
<div>
<p>McDonald, D., Price, M. N., Goodrich, J., Nawrocki, E. P., DeSantis, T. Z., Probst, A., … Hugenholtz, P. (2012). An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea. <em>The ISME Journal</em>, <em>6</em>(3), 610–618. <a href="http://doi.org/10.1038/ismej.2011.139" class="uri">http://doi.org/10.1038/ismej.2011.139</a></p>
</div>
<div>
<p>McIlroy, S. J., Saunders, A. M., Albertsen, M., Nierychlo, M., McIlroy, B., Hansen, A. A., … Nielsen, P. H. (2015). MiDAS: The field guide to the microbes of activated sludge. <em>Database</em>, <em>2015</em>(2), 1–8. <a href="http://doi.org/10.1093/database/bav062" class="uri">http://doi.org/10.1093/database/bav062</a></p>
</div>
<div>
<p>Metzker, M. L. (2010). Sequencing technologies — the next generation. <em>Nature Reviews Genetics</em>, <em>11</em>(1), 31–46. <a href="http://doi.org/10.1038/nrg2626" class="uri">http://doi.org/10.1038/nrg2626</a></p>
</div>
<div>
<p>Minchin, P. R. (1987). An evaluation of the relative robustness of techniques for ecological ordination. <em>Theory and Models in Vegetation Science</em>, 89–107. <a href="http://doi.org/10.1007/978-94-009-4061-1_9" class="uri">http://doi.org/10.1007/978-94-009-4061-1_9</a></p>
</div>
<div>
<p>Mino, T., Loosdrecht, M. C. M. van, &amp; Heijnen, J. J. (1998). Review paper - Microbiology and biochemistry of the enhanced biological phosphate removal process. <em>Wat. Res.</em>, <em>11</em>(32), 3193–3207.</p>
</div>
<div>
<p>Mullis, K., Faloona, F., Scharf, S., Saiki, R., Horn, G., &amp; Erlich, H. (1986). Specific Enzymatic Amplification of DNA In Vitro: The Polymerase Chain Reaction. <em>Cold Spring Harbor Symposia on Quantitative Biology</em>, <em>51</em>(1), 263–273. <a href="http://doi.org/10.1101/SQB.1986.051.01.032" class="uri">http://doi.org/10.1101/SQB.1986.051.01.032</a></p>
</div>
<div>
<p>Nielsen, P. H., &amp; McMahon, K. D. (2014). <em>CHAPTER 2. MICROBIOLOGY AND MICROBIAL ECOLOGY OF THE ACTIVATED SLUDGE PROCESS</em>. IWA Publishing. Retrieved from <a href="https://books.google.dk/books?id=J7cDBAAAQBAJ" class="uri">https://books.google.dk/books?id=J7cDBAAAQBAJ</a></p>
</div>
<div>
<p>Nielsen, P. H., Mielczarek, A. T., Kragelund, C., Nielsen, J. L., Saunders, A. M., Kong, Y., … Vollertsen, J. (2010). A conceptual ecosystem model of microbial communities in enhanced biological phosphorus removal plants. <em>Water Research</em>, <em>44</em>(17), 5070–5088. <a href="http://doi.org/10.1016/j.watres.2010.07.036" class="uri">http://doi.org/10.1016/j.watres.2010.07.036</a></p>
</div>
<div>
<p>Orhon, D. (2015). Evolution of the activated sludge process: the first 50 years. <em>Journal of Chemical Technology &amp; Biotechnology</em>, <em>90</em>(4), 608–640. <a href="http://doi.org/10.1002/jctb.4565" class="uri">http://doi.org/10.1002/jctb.4565</a></p>
</div>
<div>
<p>Pearson, K. (1901). On lines and planes of closest fit to systems of points in space. <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em>, <em>2</em>(1), 559–572. <a href="http://doi.org/10.1080/14786440109462720" class="uri">http://doi.org/10.1080/14786440109462720</a></p>
</div>
<div>
<p>Podani, J., &amp; Miklós, I. (2002). Resemblance Coefficients and the Horseshoe Effect in Principal Coordinates Analysis. <em>Ecology</em>, <em>83</em>(12), 3331–3343. Retrieved from <a href="http://www.jstor.org/stable/3072083" class="uri">http://www.jstor.org/stable/3072083</a></p>
</div>
<div>
<p>Qin, D., Abdi, N. M., &amp; Fredrick, K. (2007). Characterization of 16S rRNA mutations that decrease the fidelity of translation initiation. <em>RNA (New York, N.Y.)</em>, <em>13</em>(12), 2348–55. <a href="http://doi.org/10.1261/rna.715307" class="uri">http://doi.org/10.1261/rna.715307</a></p>
</div>
<div>
<p>Quast, C., Pruesse, E., Yilmaz, P., Gerken, J., Schweer, T., Yarza, P., … Glockner, F. O. (2013). The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. <em>Nucleic Acids Research</em>, <em>41</em>(D1), D590–D596. <a href="http://doi.org/10.1093/nar/gks1219" class="uri">http://doi.org/10.1093/nar/gks1219</a></p>
</div>
<div>
<p>Quinn, G. R., &amp; Skerman, V. B. D. (1980). Herpetosiphon—Nature’s scavenger? <em>Current Microbiology</em>, <em>4</em>(1), 57–62. <a href="http://doi.org/10.1007/BF02602893" class="uri">http://doi.org/10.1007/BF02602893</a></p>
</div>
<div>
<p>Ramette, A. (2007). Multivariate analyses in microbial ecology. <em>FEMS Microbiology Ecology</em>, <em>62</em>(2), 142–60. <a href="http://doi.org/10.1111/j.1574-6941.2007.00375.x" class="uri">http://doi.org/10.1111/j.1574-6941.2007.00375.x</a></p>
</div>
<div>
<p>Sanger, F., &amp; Coulson, A. (1975). A rapid method for determining sequences in DNA by primed synthesis with DNA polymerase. <em>Journal of Molecular Biology</em>, <em>94</em>(3), 441–448. <a href="http://doi.org/10.1016/0022-2836(75)90213-2" class="uri">http://doi.org/10.1016/0022-2836(75)90213-2</a></p>
</div>
<div>
<p>Sanger, F., Nicklen, S., &amp; Coulson, A. R. (1977). DNA sequencing with chain-terminating inhibitors. <em>Proceedings of the National Academy of Sciences</em>, <em>74</em>(12), 5463–5467. <a href="http://doi.org/10.1073/pnas.74.12.5463" class="uri">http://doi.org/10.1073/pnas.74.12.5463</a></p>
</div>
<div>
<p>Saunders, A. M., Albertsen, M., Vollertsen, J., &amp; Nielsen, P. H. (2016). The activated sludge ecosystem contains a core community of abundant organisms. <em>The ISME Journal</em>, <em>10</em>(1), 11–20. <a href="http://doi.org/10.1038/ismej.2015.117" class="uri">http://doi.org/10.1038/ismej.2015.117</a></p>
</div>
<div>
<p>Seviour, R., &amp; Nielsen, P. H. (2010). <em>Microbial Ecology of Activated Sludge</em>. IWA Publishing. Retrieved from <a href="https://books.google.dk/books?id=z9ohFDYcAsgC" class="uri">https://books.google.dk/books?id=z9ohFDYcAsgC</a></p>
</div>
<div>
<p>Shepard, R. N. (1966). Metric structures in ordinal data. <em>Journal of Mathematical Psychology</em>, <em>3</em>(2), 287–315. <a href="http://doi.org/10.1016/0022-2496(66)90017-4" class="uri">http://doi.org/10.1016/0022-2496(66)90017-4</a></p>
</div>
<div>
<p>Valenzuela-Encinas, C., Neria-González, I., Alcántara-Hernández, R. J., Estrada-Alvarado, I., Zavala-Díaz de la Serna, F. J., Dendooven, L., &amp; Marsch, R. (2009). Changes in the bacterial populations of the highly alkaline saline soil of the former lake Texcoco (Mexico) following flooding. <em>Extremophiles</em>, <em>13</em>(4), 609–621. <a href="http://doi.org/10.1007/s00792-009-0244-4" class="uri">http://doi.org/10.1007/s00792-009-0244-4</a></p>
</div>
<div>
<p>Whittaker, R. H. (1967). Gradient Analysis of Vegetation*. <em>Biological Reviews</em>, <em>42</em>(2), 207–264. <a href="http://doi.org/10.1111/j.1469-185X.1967.tb01419.x" class="uri">http://doi.org/10.1111/j.1469-185X.1967.tb01419.x</a></p>
</div>
<div>
<p>Whittaker, R. H. (1972). Evolution and Measurement of Species Diversity. <em>Taxon</em>, <em>21</em>(2/3), 213–251. Retrieved from <a href="http://www.jstor.org/stable/1218190" class="uri">http://www.jstor.org/stable/1218190</a></p>
</div>
<div>
<p>Woese, C. R., &amp; Fox, G. E. (1977). Phylogenetic structure of the prokaryotic domain: The primary kingdoms. <em>Proceedings of the National Academy of Sciences</em>, <em>74</em>(11), 5088–5090. <a href="http://doi.org/10.1073/pnas.74.11.5088" class="uri">http://doi.org/10.1073/pnas.74.11.5088</a></p>
</div>
<div>
<p>Zhang, T., Shao, M.-F., &amp; Ye, L. (2012). 454 Pyrosequencing reveals bacterial diversity of activated sludge from 14 sewage treatment plants. <em>The ISME Journal</em>, <em>6</em>(6), 1137–1147. <a href="http://doi.org/10.1038/ismej.2011.188" class="uri">http://doi.org/10.1038/ismej.2011.188</a></p>
</div>
</div>
</div>









            </section>

          </div>
        </div>
      </div>
<a href="C-WWTPcharacteristics.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
